test/functional/lua/snippet_spec.lua

```diff
@@ -230,14 +231,29 @@ describe('vim.snippet', function()
   end)
 
   it('updates snippet state when built-in completion menu is visible', function()
-    test_expand_success({ '$1 = function($2)\n$3\nend' }, { ' = function()', '', 'end' })
+    local curbuf = api.nvim_get_current_buf()
+    api.nvim_set_option_value('sw', 2, { buf = curbuf })
```

Comment:
Look at the signature of `test_expand_success`. You can pass a settings parameter instead of setting the options like this.
https://github.com/neovim/neovim/blob/e1af3817958a2b9a818fcb10eec9f1197bd62dfd/test/functional/lua/snippet_spec.lua#L71

Reply:
oh right. i saw it :)